FRW/Modules/Wallet/TokenDetail/TokenDetailView.swift

```diff
@@ -339,6 +341,40 @@ struct TokenDetailView: RouteableView {
         }
         .padding(.vertical, 7)
     }
+    
+    var storageView: some View {
+        HStack(spacing: 0) {
+            StorageUsageView(
+                usage: $vm.storageUsageDesc,
+                usagePercentValue: $vm.storageUsagePercent
+            )
+            .headerView(
+                HStack {
+                    Text("storage_usage".localized)
+                        .font(.inter(size: 16, weight: .semibold))
+                    
+                    Spacer()
+                    
+                    Text(String(format: "%.3f FLOW", vm.storageFlow))
+                }
+            )
+            .footerView(
+                HStack {
```

Comment:
Missed a separate line

Reply:
Sorry not sure what you mean...

Reply:
Yeah my bad... it was barely visible when zoomed out